README.md

```diff
@@ -5,4 +5,26 @@
 
 # Google Container Tools for IntelliJ
 
-In development
+This plugin integrates continuous development on [Kubernetes](https://www.kubernetes.io) using [Skaffold](https://skaffold.dev) into [Jetbrains family of IDEs](https://www.jetbrains.com/products.html), including IntelliJ (both Community and Ultimate editions), GoLand, PyCharm and WebStorm.
+
+**Note**: This plugin is still in development.
+
+## Features
+
+* One click deployment to Kubernetes clusters right from your IDE using kubectl, Helm, Google Cloud Build, Jib and Kanico using Skaffold integration with IDE.
+* Continuous development on Kubernetes. Watches the dependencies of your docker image or Jib Java project for changes, so that on any change, Skaffold builds and deploys your application to a Kubernetes cluster.
+* Automatic discovery and support for project with existing Skaffold configuration, in any language supported by your preferred Jetbrains IDE.
+* Initial support for Skaffold configuration files editing and smart templates.
+
+## Prerequisites and required dependencies
+
+This plugin relies on existing Kubernetes and container tools and libraries to provide rich Kubernetes experience built into your IntelliJ or other Jetbrains IDE. At the minimum, the following tools are expected to be installed and setup on your system and available in the system path:
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for working with Kubernetes clusters and managing Kubernetes deployments.
+* [Skaffold](https://skaffold.dev/docs/getting-started/) to support continuous development on a Kubernetes cluster, smart image building and tagging, and an array of supported deployment and build types.
+* [Docker](https://www.docker.com/) for building and pushing your container images. *Note*: Docker is optional if you are using [Jib to build container image](https://github.com/GoogleContainerTools/jib) together with a Maven/Gradle project for your Java IntelliJ IDE and push to a non-Docker container repository.
+* Configured Kubernetes cluster. It could be a cluster for local development, such as [Minikube](https://kubernetes.io/docs/setup/minikube/) or [Docker Kubernetes](https://docs.docker.com/docker-for-mac/kubernetes/) cluster, or remote cluster, such as [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) cluster. We recommend [Minikube](https://kubernetes.io/docs/setup/minikube/) cluster for local development.
+
+## Installing the plugin into your IDE
+
+## Getting started
```

Comment:
perhaps add a `(coming soon)` to the sections that we aren't filling out yet?

Reply:
just an idea - but we could (later) expand this section to show steps / screenshots using the plugin with your (not-yet-merged) hello-world sample

Reply:
Exactly, I have the same idea in mind! Guided getting started with the getting started example from our repo. Already working on this and collecting screenshots, this is going to be next PR for docs.